app/controllers/books_controller.rb

```diff
@@ -10,4 +10,15 @@ def show
   def index
     render_paginated Book.all
   end
+
+  def info_book
+    isbn = params[:isbn]
+    return render json: { error: t('.no_isbn') }, status: :bad_request if isbn.nil?
```

Comment:
Where is this translation defined?

Reply:
Also, you can use strong parameters and avoid checking that the parameter is being sent.
Some method similar to this:
```ruby
def book_info_params
  params.require(:isbn)
end
```